benches/README.md

```diff
@@ -0,0 +1,55 @@
+## Context
+
+There are two primary flows that we want to be able to report estimated times on for Forte: key generation and signing. The threshold ECDSA library breaks down these flows into two steps each: key generation includes keygen and aux-info; signing includes pre-signing and signing. 
```

Comment:
todo: Don't mention client names in public repos. 

More broadly, please rewrite this intro for the more general audience of cryptographic engineers who might use this. The benchmarks are just for the subprotocols and don't necessarily have to be bundled into two flows; other use cases might do actual offline presigning separately from signing, for example.